packages/browser-wallet/src/background/credential-deployment.ts

```diff
@@ -42,6 +42,10 @@ async function createAndSendCredential(credIn: CredentialInputV1): Promise<Crede
 
     // Add Pending
     await addCredential(newCred);
+    // Set Selected to new account
+    await storedSelectedAccount.set(address);
+    // Remove guard stopping another credential being created
+    await sessionCreatingCredential.set(false);
```

Comment:
I think that the removal of the guard has to be in a finally clause, e.g. if the deployment is not successful, then the guard is not reset.